week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -15,18 +25,30 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+
+  if (num === num && num % 2 === 0 && num <= 100) {
+
+    return true;
+  }
+
+  else {
+    return false;
+  }
+}
 
 /* 
-Write a function that removes an element from an array
+Write a function that removes an element from an aray
 The function must:
 - NOT change the original array
 - return a new array with the item removed
 - remove the item at the specified index
 */
 
 function remove(arr, index) {
-  return; // complete this statement
+  arr.splice(index, 1);
```

Comment:
So there is a mistake here. while your fix will pass, but it won't follow the condition (not change original array)
try to search splice destructive js
and try to do it using slice (twice maybe)

Reply:
function remove(arr, index) {
  newArray = arr.slice(0);           //copying the arr
  newArray.splice(index, 1);
  console.log(newArray);
  return newArray;